src/rt/backtrace/elf.d

```diff
@@ -52,6 +52,10 @@ struct Image
         if (dbgSectionIndex != -1)
         {
             auto dbgSectionHeader = ElfSectionHeader(&file, dbgSectionIndex);
+            // we don't support compressed debug sections
+            static if (__traits(compiles, SHF_COMPRESSED))
```

Comment:
AFAICT there's no reason for this special casing here. FreeBSD and DragonflyBSD also use the same ELF format and thus imho it's better to add the `SHF_COMPRESSED` constant to the respective files.